media-gfx/povray/povray-3.7.0.0-r2.ebuild

```diff
@@ -0,0 +1,143 @@
+			ax_compiler_version.m4 || die
+		popd &>/dev/null || die
+	bash -x prebuild.sh || die
+	popd &>/dev/null || die
+
+	rm -rf libraries || die
+
+	# Change some directories that cannot be adjusted via configure
+	sed \
+		-e 's:${povsysconfdir}/$PACKAGE/$VERSION_BASE:${povsysconfdir}/'${PN}':g' \
+		-e 's:${povdatadir}/$PACKAGE-$VERSION_BASE:${povdatadir}/'${PN}':g' \
+		-e 's:${povdatadir}/doc/$PACKAGE-$VERSION_BASE:${povdatadir}/doc/'${PF}':g' \
+		-e 's:BOOST_THREAD_LIBS $LIBS:BOOST_THREAD_LIBS $LIBS -lboost_date_time:g' \
+		-e 's:"/usr/include":"'${EPREFIX}'/usr/include":' \
+		-e 's:"/usr/lib":"'${EPREFIX}'/usr/'$(get_libdir)'":' \
+		-i configure.ac || die
+
+	sed \
+		-e "s:^povlibdir = .*:povlibdir = @datadir@/${PN}:" \
+		-e "s:^povdocdir = .*:povdocdir = @datadir@/doc/${PF}:" \
+		-e "s:^povconfdir = .*:povconfdir = @sysconfdir@/${PN}:" \
+		-e 's:mkdir_p:MKDIR_P:g' \
+		-i Makefile.am || die
+
+	# The "+p" option on the test command line causes a pause and
+	# prompts the user to interact, so remove it.
+	sed -i -e "s:biscuit.pov -f +d +p:biscuit.pov -f +d:" Makefile.am || die
+	default
+
+	eautoreconf
+}
+
+src_configure() {
+	# Fixes bug 71255
+	if [[ $(get-flag march) == k6-2 ]]; then
+		filter-flags -fomit-frame-pointer
+	fi
+
+	# The config files are installed correctly (e.g. povray.conf),
+	# but the code compiles using incorrect [default] paths
+	# (based on /usr/local...), so povray will not find the system
+	# config files without the following fix:
+	append-cppflags -DPOVLIBDIR=\\\"${EROOT}usr/share/${PN}\\\" -DPOVCONFDIR=\\\"${EROOT}etc/${PN}\\\"
+
+	# TODO: Restore OpenEXR if upstream start to support OpenEXR 3/imath
+	econf \
+		COMPILED_BY="Portage (Gentoo $(uname)) on $(hostname -f)" \
+		$(use_enable debug) \
+		$(use_enable io-restrictions) \
+		--without-openexr \
+		$(use_with tiff libtiff "${EPREFIX}/usr/$(get_libdir)") \
+		$(use_with X libsdl "${EPREFIX}/usr/$(get_libdir)") \
+		$(use_with X x "${EPREFIX}/usr/$(get_libdir)") \
+		$(usex tiff "" "NON_REDISTRIBUTABLE_BUILD=yes") \
+		--with-boost-libdir="${EPREFIX}/usr/$(get_libdir)" \
+		--without-libmkl \
+		--disable-pipe \
+		--disable-static \
+		--disable-strip \
+		--disable-optimiz \
+		--disable-optimiz-arch
+}
+
+src_test() {
+	virtx default
+}
+
+pkg_preinst() {
+	# Copy the old config files if they are in the old location
+	# but do not yet exist in the new location.
+	# This way, they can be treated by CONFIG_PROTECT as normal.
+	local conf_file version_dir
+	for conf_file in "${ED}"/etc/"${PN}"/*; do
+		if [ ! -e "${EROOT}etc/${PN}/${conf_file}" ]; then
+			for version_dir in $(echo "${EROOT}"etc/"${PN}"/* | grep "^[0-9]" | sort -rn); do
+				if [ -e "${EROOT}etc/${PN}/${version_dir}/${conf_file}" ]; then
+					mv "${EROOT}etc/${PN}/${version_dir}/${conf_file}" "${EROOT}etc/${PN}" || die
+					elog "Note: ${conf_file} moved from ${EROOT}etc/povray/${version_dir}/ to ${EROOT}etc/povray/"
+					break
+				fi
+			done
+		fi
+	done
+}
```

Comment:
Would bumping to 3.8.0 work with OpenEXR-3?

Reply:
@SoapGentoo

Reply:
Yes, I was probably thinking of that version. Saw this a few months ago, and thought it's already released.